manifests/m/Mango3D/LycheeSlicer/7.3.1/Mango3D.LycheeSlicer.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Mango3D.LycheeSlicer
+PackageVersion: 7.3.1
+InstallerType: nullsoft
+Scope: machine
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+Protocols:
+- lycheeslicer
+FileExtensions:
+- 3mf
+- cws
+- lstl
+- lys
+- lyt
+- obj
+- stl
+ProductCode: 8a7f68fc-0ecf-58b3-94e0-fb6e1ea8d905
+ReleaseDate: 2025-03-26
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mango-lychee.nyc3.digitaloceanspaces.com/LycheeSlicer-7.3.1.exe
+  InstallerSha256: 07C89A12451916753EDCCA750117B1E503854472FC662B62E5B18C28AB0A9CA9

```

```suggestion
  InstallerSha256: 2CF59F60F07EA356D23B823493B2DB28D6F26B8705CE2F9D00497741DC8B9EAB
```

(Automated response - build 1026.)